assets/xml/objects/object_zod.xml

```diff
@@ -10,10 +10,10 @@
         <!-- Tijo DisplayLists -->
         <DList Name="gTijoLowerBodyChairDL" Offset="0x39A0" />
         <DList Name="gTijoUpperBodyDL" Offset="0x3DA0" />
-        <DList Name="gTijoUpperRightArmDL" Offset="0x4018" />
+        <DList Name="gTijoRightForearmDL" Offset="0x4018" />
         <DList Name="gTijoLowerRightArmDL" Offset="0x4130" />
         <DList Name="gTijoRightHandDL" Offset="0x4258" />
-        <DList Name="gTijoUpperLeftArmDL" Offset="0x43E8" />
+        <DList Name="gTijoLeftForearmDL" Offset="0x43E8" />
```

Comment:
The forearm is the part of the arm below the elbow, so think you changed the wrong ones.